DigitalLearningSolutions.Web/Services/RegistrationService.cs

```diff
@@ -200,16 +203,33 @@ IGroupsService groupsService
 
             try
             {
+                var emailUpdate = new PossibleEmailUpdate

```

Comment:
Can you please rename all the `var emailUpdate` to `var possibleEmailUpdate`, since out of context (i.e. where it is passed into another method) it can be a bit unclear what this thing is for

Reply:
`emailUpdate` is still present in:

- DigitalLearningSolutions.Data/DataServices/RegistrationDataService.cs
- DigitalLearningSolutions.Web/Services/EmailVerificationService.cs
- DigitalLearningSolutions.Web/Services/RegistrationService.cs

Can you rename it there too please?